packages/container/tests/container_test.js

```diff
@@ -535,3 +517,27 @@ QUnit.test('Lazy injection validations are cached', function() {
   container.lookup('apple:main');
   container.lookup('apple:main');
 });
+
+if (isEnabled('ember-container-inject-owner')) {
+  QUnit.test('A deprecated `container` property is appended to every instantiated object', function() {
+    const registry = new Registry();
```

Comment:
this should likely be `let`

`const` to express variable bindings that should ABSOLUTELY not change, not bindings that happen to not change
`let` express variable bindings that may or may not change.

The goal of this difference as señor @wycats explain to me, is to not overly abuse `const`, doing so will likely desensify devs and diminish its value.